cinema/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class CinemaConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "cinema"
```

Comment:
The 'name' attribute is correctly set to 'cinema', which is necessary for Django to recognize the app configuration.